charts/icm-web/templates/ingress.yaml

```diff
@@ -25,7 +24,7 @@ spec:
         {{- range .hosts }}
         - {{ . | quote }}
         {{- end }}
-      secretName: {{ .Values.ingress.secretName | default (printf "%s-wa-cert" (include "icm-web.fullname" .)) }}
+      secretName: {{ .secretName }}
```

Comment:
The secret is still outside of the hosts-loop so all hosts will share the same secrets. As mentioned earlier this will not work.

Reply:
but the secret is within the tls loop
This then could look like:
```
  tls:
  - hosts:
    - domain_1
    secretName: cert_1
  - hosts:
    - domain_2
    secretName: cert_2
```